internal/controller/reconcile-capapplicationversion.go

```diff
@@ -618,8 +647,8 @@ func getContainer(params *DeploymentParameters) []corev1.Container {
 		ImagePullPolicy: params.WorkloadDetails.DeploymentDefinition.ImagePullPolicy,
 		Command:         params.WorkloadDetails.DeploymentDefinition.Command,
 		Env:             getEnv(params),
```

Comment:
We also need to enhance the env in this case to add `SERVICE_BINDING_ROOT` in the xsenv scenarios.